source/user/sgx.rst

```diff
@@ -0,0 +1,56 @@
+.. _user_sgx:
+
+Seminumerical Exchange (SGX)
+****************************
+
+*Modules*: :mod:`sgx`
+
+The SGX module implements seminumerical computation of the exchange matrix.
+
+Introduction
+============
+
+Direct computation of the Hartree-Fock exchange matrix in the atomic orbital basis scales poorly with system size.
+To achieve better scaling, one three-dimensional integral in the 6-dimensional two-electron integrals can be computed analytically, while the other can be evaluated on a real-space grid, as proposed by Friesner :cite:`Friesner1985`.
+The PySCF implementation resembles the chain-of-spheres (COSX) algorithm of Neese et al. :cite:`Neese2009`, but uses more conservative grids and does not implement P-junction screening.
+Overlap fitting is used to reduce aliasing errors :cite:`Izsak2011`.
+SGX scales as :math:`O(N^2)` with system size, as opposed to the :math:`O(N^4)` scaling of analytical exchange :cite:`Neese2009`.
+
+Usage and Example
+=================
+
+Any :attr:`scf.hf.SCF` object :attr:`mf` can be converted to an equivalent object that computes the Coulomb and Exchange matrices with SGX instead of analytical integration by calling :code:`sgx.sgx_fit(mf)`.
+
+* :source:`examples/sgx/00-simple_sgx.py`
+
+.. literalinclude:: ../../examples/sgx/00-simple_sgx.py
+
+.. code::
+
+  converged SCF energy = -76.0267374704185
+  converged SCF energy = -76.0267978618974
+
+In this case, the error of SGX compared to analytical exchange is about 0.06 mEh. The line
+
+.. code::
+
+    mf.with_df.dfj = True
+
+specifies to compute the Coulomb matrix using Density Fitting (DF) while using SGX for the Exchange matrix.
```

Comment:
``suggestion
specifies to compute the Coulomb matrix using density fitting (DF-J) while using SGX for the exchange matrix.
```